website/src/lib/oasst_api_client.ts

```diff
@@ -313,8 +313,8 @@ export class OasstApiClient {
     return this.get<Message[]>(`/api/v1/messages?${params}`);
   }
 
-  fetch_recent_messages() {
-    return this.get<Message[]>(`/api/v1/messages`);
+  fetch_recent_messages(lang: string) {
+    return this.get<Message[]>(`/api/v1/messages?lang=${lang}`);
```

Comment:
you can pass an object as a 2nd param of the get method here
```suggestion
    return this.get<Message[]>(`/api/v1/messages`, { lang });
```

Reply:
Done!